sdk/spring/azure-spring-data-cosmos/src/main/java/com/azure/spring/data/cosmos/common/CosmosSchedulers.java

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.spring.data.cosmos.common;
+
+import reactor.core.scheduler.Scheduler;
+import reactor.core.scheduler.Schedulers;
+
+/**
+ * Schedulers for azure-spring-data-cosmos
+ */
+public class CosmosSchedulers {
```

Comment:
We should make this class `final`

Reply:
Actually, we really don't need to expose this class at all. Why don't we make this a package private class in the core package itself, since we are not using it outside of the core package anyway.

Reply:
Done.